### ISSUE TITLE
Additional cooler features

### ISSUE THREAD

**Issue by jessakay** [deeptools/pyGenomeTracks#245]
Thank you for maintaining and continuously improving this great library! I'm now attempting to generate some figures involving Hi-C and has run into a few minor hiccups.

---
All of our data are stored as mcool files and so I initially tried to specify the complete URI in the configuration file but received the following error:

`FileNotFoundError: [Errno 2] No such file or directory: '/test.mcool::/resolutions/10000'`

It appearrs that while HiCMatrix correctly handles such a path, 

https://github.com/deeptools/pyGenomeTracks/blob/c5a24faebe8c8cf754926e922250c1e41b6859cb/pygenometracks/tracksClass.py#L535

is not aware of mcool files' structure. It would be great to avoid the minor inconvenience of needing to separately producing a copy of single-resolution cooler files.

---

As previously discussed in https://github.com/deeptools/HiCExplorer/issues/216, balancing weights from cooler files are scaled such that the normalized values fall within [0, 1], and so can sometimes cause issues with the typical `log1p` transformation. However, it is possible the obtain values on the original scale through a scaling factor:

```
import cooler
c = cooler.Cooler('test.mcool::/resolutions/10000')
c._load_attrs('bins/weight')['scale']
```

Which can be a single scalar or a vector depending on if balancing was performed genome-wide or per chromosome.

Having the option to enable the multiplication such scaling factors would be really helpful.

---

pyGenomeTracks version: 3.4
python version: 3.7.7

**Comment by lldelisle** [comment]
Hi,
Thanks for the bug report. For the part 1 this is on going in #247 , for the part 2 I am not aware of this option in cool and not sure this can be easily implemented. However, I don't know if you are aware of the `scale_factor` which is used before the log1p transformation.

**Comment by lldelisle** [comment]
Hi,
mcool is now available on version 3.5.

**Comment by jessakay** [comment]
Thank you! Works great
